i18n/en_GB/messages/messages.json

```
@@ -530,6 +530,16 @@
   "update_address_page_title": "Update Address",
   "use_of_funds_page_title": "Use Of Funds",
 
-  "language_selection": "Open language selection"
-
+  "language_selection": "Open language selection",
+
+  "donation_form_summary_title": "Summary",
+  "donation_form_payment_headline": "You will donate:",
```

calling it "payment headline" could be too ambiguous, because the "payment section" of the donation form is somewhere else

```suggestion
  "donation_form_summary_headline": "You will donate:",
```